ocp_resources/network_attachment_definition.py

```diff
@@ -24,4 +46,149 @@ def to_dict(self):
                 f"{NamespacedResource.ApiGroup.K8S_V1_CNI_CNCF_IO}/resourceName": self.resource_name
             }
         res["spec"] = {}
+        if self.config:
+            res["spec"]["config"] = self.config
+        else:
+            res["spec"]["config"] = {
+                "cniVersion": self.cni_version,
+                "type": self.cni_type,
+            }
+        return res
+
+
+class BridgeNetworkAttachmentDefinition(NetworkAttachmentDefinition):
+    def __init__(
+        self,
+        name,
+        namespace,
+        bridge_name,
+        cni_type,
```

Comment:
This does not have a default now?

Reply:
I thought that `bridge` is the default, based on the class.

Reply:
this is for other bridges
if user want to work with linux-bridge or OVS he should call OvsBridgeNetworkAttachmentDefinition or LinuxBridgeNetworkAttachmentDefinition